src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/PublicDataRetrievalService.java

```diff
@@ -32,13 +36,18 @@ public OfficersApi getOfficerApi() {
         return officers;
     }
 
+    public PscsApi getPscsApi() {
```

Comment:
remove Api from method name

Reply:
Should the getOfficerApi() also be changed to getOfficerApi() ?

Reply:
I just took the standard pattern for getters. They just need to use the same convention I'd say (getCompanyProfile is different). I'd prefer no Api